packages/suite-desktop-core/e2e/tests/trezor-connect/connect-ws.test.ts

```diff
@@ -86,6 +86,26 @@ test.describe('TrezorConnect', { tag: ['@group=suite', '@desktopOnly'] }, () =>
 
         expect(await res).toMatchObject({ success: true });
     });
+
+    test('TrezorConnect.ethereumSignMessage', async ({ connectModal, trezorUserEnvLink, page }) => {
+        const res = TrezorConnect.ethereumSignMessage({
+            path: "m/44'/60'/0'",
+            message: 'example message',
+        });
+
+        await expect(connectModal.header).toHaveText('Sign Ethereum message');
+        await passThroughPermissions(connectModal);
```

Comment:
```suggestion
        await passThroughPermissions(connectModal);
        await expect(connectModal.header).toHaveText('Sign Ethereum message');
```

It's necessary to switch the order due to new design (method name is not included in permissions modal)